### ISSUE TITLE
Support of LSC Smart Flood Light possible ?

### ISSUE THREAD

**Issue by Tottback** [Apollon77/ioBroker.tuya#406]
Is it possible to support also the "LSC LED Smart Flood Light" ?
[](https://www.action.com/de-de/p/lsc-smart-connect-led-aubenstrahler/ )

I could not add the device to the old Smartlife-App (3.12.6), therefore I couldn't try to integrate it to the Tuya-Adapter by proxy.
Could I add the device with an hidden/compatible device type to the SmartLife-App?
With LSC App no problem, I get also an virtual ID (bfxxxxxxx)
Or any other way for integration ?

**Comment by Apollon77** [comment]
No, no other way for now but planned, just need to find time

**Comment by Ilovegym66** [comment]
Hi, I'd it working, latest Adapter Version from Git ( 4 Weeks ago ) and add it with the MEMU Android 6 Simulator.
I'd add it with the current version from my iphone with the smartlife App, and than I started the Adapter Proxy and MEMU.

![image](https://user-images.githubusercontent.com/36229898/164628760-74142397-bfe6-40bf-89cb-7f53405603b1.png)

**Comment by Apollon77** [comment]
"iphon smartlife App" will be simply too new version ... yopu need to do the proxy Sync with that Anrdoid Simulataor again

**Comment by Ilovegym66** [comment]
exactly, it's needed, because the lamp can't add in the old version of the smartlife app. I make this in general, first add the device to wifi / smartlife with the current version, than I changed the ip address in my dhcp server for it, reboot the to-add-device, that it get the new ip, than I close the iphone app, start the emulator, start the proxy and zack, it's inlcuded...

**Comment by Tottback** [comment]
Thanks for information. 
Hmm, I also got an IP for the Tuya Flood Light by usage of the LSC Smart Connect App.
Unfortunally my Router (Telekom Hybrid) doesn't allow to switch that MAC-address afterwards to another IP. 
Even if this would be possible, I'm not sure, to which IP I would had to change. How can the old App get aware of that device ?

**Comment by Apollon77** [comment]
> How can the old App get aware of that device ?

Just Log in into the same account and done

**Comment by Tottback** [comment]
OK, then I have to use the new version of the Smartlife App on another device to integrate the FloodLight instead of the "LSC Smart Connect" App, because both apps are probably not connected to each other.

**Comment by Tottback** [comment]
I didn't find a suitable device even with SmartLife App 3.38.0. Which one did you chose?

**Comment by Tottback** [comment]
I added the Flood Light as Lightsource (WiFi), it was recognized then as Smart Floodlight with all features.
I also could add it to Tuya-Adapter (I had to reinstall the Tuya-Adapter to get all devices whyever).
The changes in the objects occurs, when I change settings by the App, although "Device Online Status" is shown as "false", but I couldn't change the settings by the objects (also when App is closed). 
After restart of Tuya-Adapter "Device Online Status" was "true" and now activation by objects works. Thanks
